presentation/diaryfeedback/src/main/java/com/hilingual/presentation/diaryfeedback/DiaryFeedbackUiState.kt

```diff
@@ -0,0 +1,41 @@
+package com.hilingual.presentation.diaryfeedback
+
+import androidx.compose.runtime.Immutable
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.persistentListOf
+
+@Immutable
+data class DiaryFeedbackUiState(
+    val diaryId: Long = 0L,
+    val writtenDate: String = "",
+    val isAI: Boolean = true,
```

Comment:
p2
위에서 코리 반영했던 것처럼 여기도 네이밍 맞춰주면 좋을 것 같아용
```suggestion
    val isAIWritten: Boolean = true,
```